frontend/src/components/explorer/mapViewer/DataOverlay.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div id="dataOverlayBox"
-       class="column is-one-fifth-widescreen is-one-quarter-desktop
-         is-one-quarter-tablet has-background-lightgray">
+       :class="{ 'is-one-fifth-widescreen is-one-quarter-desktop is-one-quarter-tablet': position === 'absolute'}"
```

Comment:
```suggestion
       :class="{ 'is-one-fifth-widescreen is-one-quarter-desktop is-one-quarter-tablet': this.position === 'absolute'}"
```

Without the suggestion I got this:
![image](https://user-images.githubusercontent.com/23480589/139667204-04dd6c99-b9bb-4426-a09a-6dcc16d5322d.png)

Reply:
I don't have this visual problem here. Which browser is this from?

Reply:
Firefox 93. I also freshly rebuilt the stack with a new `yarn.lock`, perhaps you can try that.

Reply:
Sorry it looks like it was a cache thing - the issue has disappeared without my suggestion.